forge-gui/res/cardsfolder/upcoming/anoint_with_affliction.txt

```diff
@@ -0,0 +1,9 @@
+Name:Anoint with Affliction
+ManaCost:1 B
+Types:Instant
+A:SP$ Branch | ValidTgts$ Creature | BranchConditionSVar$ X | BranchConditionSVarCompare$ GE3 | TrueSubAbility$ Poisoned | FalseSubAbility$ NotPoisoned | SpellDescription$ Exile target creature if it has mana value 3 or less. Corrupted — Exile that creature instead if its controller has three or more poison counters.
```

Comment:
Branch is more difficult for AI

imo you can do this with 2 effects alone:
- Exile if Poisoned (CheckSVar)
followed by
- Exile if not Poisoned + cmc (CheckSVar + Present)

Reply:
probably best to keep the SpellDescription together now otherwise it'll look wrong ingame GUI